src/datacontroller.js

```diff
@@ -104,6 +110,8 @@ export default class DataController {
 		this.viewToModel.on( 'text', convertText(), { priority: 'lowest' } );
 		this.viewToModel.on( 'element', convertToModelFragment(), { priority: 'lowest' } );
 		this.viewToModel.on( 'documentFragment', convertToModelFragment(), { priority: 'lowest' } );
+
+		this.on( 'insertContent', ( evt, data ) => insertContent( this, data.batch, data.selection, data.content ) );
```

Comment:
I copied this from the composer. Can be `low` but it's hard to tell whether people will want to do something before or after. Since it's 50/50, default was my choice.